client/client.go

```diff
@@ -0,0 +1,74 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+
+	"github.com/giantswarm/microclient"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/go-resty/resty"
+)
+
+const (
+	// API endpoints of the service this client action interacts with.
+	CredentialEndpoint        = "/v4/organizations/%s/credentials/"
```

Comment:
What about `CredentialListEndpoint`?

Reply:
This endpoint is used to create and list credentials so I rather keep it like this.